test/parallel/test-https-agent-session-eviction.js

```diff
@@ -60,7 +60,7 @@ function second(server, session) {
   });
 
   // Let it fail
-  req.on('error', common.mustCall(function(err) {
+  req.once('error', common.mustCall(function(err) {
```

Comment:
Why this change? It seems like if this needs to be changed, something is wrong in the code in core?